modules/cert-manager-operator-post-installation-tasks.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-install.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-operator-post-installation-tasks_{context}"]
+= Post-installation cert-manager Operator tasks
+
+After installing the {cert-manager-operator}, you must wait for the cert-manager pods to come up and running. You can use the {cert-manager-operator} only after cert-manager pods are up and running.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You must have the OpenShift CLI (`oc`) installed.
+
+.Procedure
+
+* Verify that cert-manager pods are up and running by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n cert-manager
+----
++
+.Example output
+----
```

Comment:
Missed this earlier, but this should also have a [source,terminal] or [source,text] above it